packages/slice-machine/lib/migrate.js

```diff
@@ -37,9 +37,9 @@ const migrate = (model, info, env, write = true) => {
   if (write) {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
     const modelPath = CustomPaths(env.cwd)
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
```

Comment:
migrating to typescript should fix these issues

Reply:
done in the start script migration :)